tools-bom/pom.xml

```diff
@@ -38,33 +38,34 @@
 
 	<!-- Project Properties -->
 	<properties>
-		<log4j2.version>2.17.1</log4j2.version>
-		<slf4j.version>1.7.30</slf4j.version>
-		<javafx.version>17.0.2</javafx.version>
-		<jackson.version>2.13.3</jackson.version>
-		<junit.version>5.8.0-M1</junit.version>
-		<ztzip.version>1.14</ztzip.version>
+		<log4j2.version>2.20.0</log4j2.version>
+		<slf4j.version>2.0.6</slf4j.version>
+		<javafx.version>21-ea+5</javafx.version>
+		<jackson.version>2.14.2</jackson.version>
+		<junit.version>5.9.2</junit.version>
+		<ztzip.version>1.15</ztzip.version>
 		<testfx.version>4.0.16-alpha</testfx.version>
-		<bouncycastle.version>1.69</bouncycastle.version>
+		<bouncycastle.version>1.72</bouncycastle.version>
 		<passpol.version>0.7.0</passpol.version>
-		<protobuf.version>3.21.1</protobuf.version>
-		<grpc.version>1.44.0</grpc.version>
-		<apache.commons.io.version>2.8.0</apache.commons.io.version>
+		<protobuf.version>3.22.0</protobuf.version>
+		<grpc.version>1.53.0</grpc.version>
+		<apache.commons.io.version>2.11.0</apache.commons.io.version>
 		<apache.commons.lang3.version>3.12.0</apache.commons.lang3.version>
 		<apache.commons.codec.version>1.15</apache.commons.codec.version>
-		<apache.httpclient.version>4.5.13</apache.httpclient.version>
-		<picocli.version>4.6.1</picocli.version>
+		<apache.httpclient.version>4.5.14</apache.httpclient.version>
+		<picocli.version>4.7.1</picocli.version>
 		<i2p.eddsa.version>0.3.0</i2p.eddsa.version>
-		<orgjson.version>20210307</orgjson.version>
+		<orgjson.version>20230227</orgjson.version>
 		<novacrypto.version>2019.01.27</novacrypto.version>
-		<controlsfx.version>11.1.0</controlsfx.version>
+		<controlsfx.version>11.1.2</controlsfx.version>
 		<monocle.version>jdk-12.0.1+2</monocle.version>
 		<pragmatists.version>1.1.1</pragmatists.version>
-		<google.code.gson.version>2.8.9</google.code.gson.version>
-		<opencsv.version>5.6</opencsv.version>
+		<google.code.gson.version>2.10.1</google.code.gson.version>
+		<opencsv.version>5.7.1</opencsv.version>
```

Comment:
Since we changed a lot of versions, is there a way to validate everything still works fine ?

Reply:
All the tests that passed before pass now, more or less. The tests aren't as good as they should be, and the time it would take to fix them all would be a bit much. The version bumps seem to working well. I read through the notes on the updated dependencies, and they all looked like good updates. As for testing everything, I've sent the new version to be manually tested (as was the process before I joined the project). But if it is an issue, I can look into reverting the version bumps.